api/tests/MemberMatch/MemberMatchTests.cs

```diff
@@ -326,5 +329,156 @@ public void TestMethod9()
             Assert.AreEqual(probability, 0.8693, delta: 0.0001);
         }
 
+        [TestMethod]
+        public void TestBig1()
```

Comment:
I'm a bit confused what this test is actually doing. From what I can tell, it's reading in a file and just making sure running your algorithm produces the same file. Is that correct?